pkgs/development/libraries/yajl/default.nix

```diff
@@ -9,7 +9,7 @@ stdenv.mkDerivation rec {
     sha256 = "0f6yrjc05aa26wfi7lqn2gslm19m6rm81b30ksllpkappvh162ji";
   };
 
-  buildInputs = [ cmake ruby ];
+  nativeBuildInputs = [ cmake ];
```

Comment:
You should also remove ruby from the function arguments.

Reply:
Done, thank you for catching this!